### PR TITLE
Improved the Bezier curve plotting function in the file bezier-path.ts

### PR DESCRIPTION
**Pull request by DmitryTretyak**
Hi, I suggest improving the function of constructing Bezier curves. This will allow you to control all possible curve construction options using a single function. Please test it.

### CODE REVIEW

File: projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts [hunk 1]
```diff
@@ -12,175 +12,56 @@ export function bezierPath(
...
-
-  let firstControl: Point
-  let secondControl: Point
-
-  if (source.y < target.y) {
-    const distance = target.y - source.y
+    firstControl = {x: source.x, y: source.y}
+    secondControl = {x: target.x, y: target.y}
+
+    function calcContrPoint (
```
Comment (artem-mangilev) [inline projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts:26]: It's better to move this function outsize of bezierPath due to perf reasons.

Also you could add additional arguments to pass source and target points to match such signature:

```typescript
function calcControlPoint(
  point: Point,
  pointPosition: Position,
  source: Point,
  target: Point
)
```

then you could just call

```typescript
  const firstControl = calcControlPoint(source, sourcePosition, source, target)
  const secondControl = calcControlPoint(target, targetPosition, source, target)
```

Also should rename calcContrPoint -> calcControlPoint

File: projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts [hunk 2]
```diff
@@ -12,175 +12,56 @@ export function bezierPath(
...
-  }
-
-  path.bezierCurveTo(
-    firstControl.x, firstControl.y,
-    secondControl.x, secondControl.y,
-    target.x, target.y
-  )
 
-  return getPathData(path, source, target, firstControl, secondControl, usingPoints)
+  throw new Error('Unhandled combination of sourcePosition and targetPosition')
```
Comment (artem-mangilev) [inline projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts:64]: plese, remove error

File: projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts (hunk 1 above)
Comment (DmitryTretyak) [reply projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts:26]: 1. Renamed the function calcContrPoint -> calcControlPoint
2. Moved this function outsize of bezierPath
3. Add additional arguments "distance"

File: projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts (hunk 2 above)
Comment (DmitryTretyak) [reply projects/ngx-vflow-lib/src/lib/vflow/math/edge-path/bezier-path.ts:64]: The error has been removed

**Comment by artem-mangilev** [comment]
thanks, merged

**Pull request merged**